frontend-react/src/components/FileHandlers/FileHandler.test.tsx

```diff
@@ -133,545 +119,442 @@ const fileChangeEvent = {
 } as React.ChangeEvent<HTMLInputElement>;
 
 describe("FileHandler", () => {
-    test("renders a spinner while loading sender / organization", async () => {
```

Comment:
A lottttt of our tests right now are `async` by default, but if we're mocking the return values of our hooks (as opposed to using MSW), we can run expectations synchronously because the end result of the render will be synchronous.  Not a huge deal, but we don't need to be wrapping everything in Promises 🤷